app/controllers/application_controller.rb

```diff
@@ -1,4 +1,24 @@
 # frozen_string_literal: true
```

Comment:
Another one of these hiding away :)

Reply:
I went through the files and got rid of all of these now